module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -48,36 +40,52 @@ In addition to the links above, [Amezquita et al.](https://www.biorxiv.org/conte
-| `SingleCellExperiment`        | [`rowData()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#5_extracting_coldata_and_rowdata)          | Row Data   | Extracts and stores gene-level metadata that describes features of the `SingleCellExperiment` object        |
-| `scran`                | [`quickCluster()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Quick%20clustering)                | Quick Clustering     | Groups similar cells into clusters which are stored in the `SingleCellExperiment` object and are used for the calculation of size factors by `scran::computeSumFactors`                                |
-| `scran`                | [`computeSumFactors()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Deconvolution%20Methods)                 | Compute Sum Factors        | Returns a numeric vector of computed sum factors for each cell cluster stored in the `SingleCellExperiment` object. The cluster-based size factors are deconvolved into cell-based size factors that are stored in the `SingleCellExperiment` object and used by the `scran::normalize` function for the normalization of each cell's gene expression profile.                   |
-| `scater`               | [`normalize()`](https://www.rdocumentation.org/packages/scater/versions/1.0.4/topics/normalize)                       | Normalize      | Returns the `SingleCellExperiment` object with normalized expression values for each cell, using the size factors stored in the object  |
-
+| `SingleCellExperiment` | [`SingleCellExperiment()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_creating_singlecellexperiment_instances) | Single Cell Experiment| Creates a `SingleCellExperiment` object  |
+| `SingleCellExperiment`| [`colData()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#5_extracting_coldata_and_rowdata) | Column Data | Extracts and stores cell-level metadata that describes features of the `SingleCellExperiment` object|
+| `SingleCellExperiment`| [`rowData()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#5_extracting_coldata_and_rowdata)  | Row Data   | Extracts and stores gene-level metadata that describes features of the `SingleCellExperiment` object|
+| `SingleCellExperiment`| [`logcounts()`](https://bioconductor.org/packages/devel/bioc/vignettes/scater/inst/doc/vignette-intro.html#3_calculating_a_variety_of_expression_values)| Log counts| Stores or extracts log-transformed single-cell experiment count data as an assay of the `SingleCellExperiment` object|
```

Comment:
Looks like this link is broken.

Reply:
It apparently does exist anymore! :( Replaced it.